web-registry/src/components/organisms/BridgableEcocreditsTable/BridgableEcocreditsTable.tsx

```diff
@@ -76,96 +85,107 @@ export const BridgableEcocreditsTable = ({
   }
 
   return (
-    <WithLoader isLoading={isLoadingCredits} sx={loaderStyles.withLoaderBlock}>
```

Comment:
Just wrapping this in a fragment to include `<BridgeFlow>`.
Only real change starts line 184.